### ISSUE TITLE
Carbon v9 File Uploader Button shows accessibility issues with DAP tool

### ISSUE THREAD

**Issue by SowmyaRaviKrishnan** [carbon-design-system/carbon#4644]
Carbon v9 File Uploader Button shows accessibility issues with DAP tool
1.Each form control must have associated label.
2.INPUT LABEL references must be valid
3.The WAI-ARIA role(s) and/or attribute(s) must be valid for the relevant element.

## Environment

> Operating system - Linux RHEL

> Browser -Chrome

> Assistive technology used to verify -IBM DAP

## Detailed description

> What version of the Carbon Design System are you using? - v9

> What WCAG 2.1 checkpoint does the issue violate?
IBM 4.1.2 Name, Role, Value
IBM 1.3.1 Info and Relationships

**Comment by SowmyaRaviKrishnan** [comment]
@joshblack @amuqeetkhan

**Comment by abbeyhrt** [comment]
Hi @SowmyaRaviKrishnan, thanks for bringing this to our attention! As I've mentioned in the other issues for v9, we will not be backporting WCAG 2.1 fixes to v9 but we always accept community contributions for any v9 issues! If you would like this to be addressed, feel free to open up a PR and we’ll review it! 👍
